### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR with Chrome version 131

### DISCUSSION THREAD

**Discussion by truthz03** [nginx/nginx#334] [General]
Hi,

today my Chrome browser has updated from version 130 to 131.
Now my web server with nginx is not reachable anymore. I always get "ERR_SSL_PROTOCOL_ERROR"

The web server is a Windows Server 2019 maschine and I use the lastest nginx for Windows version (1.27.2)
Disabling the "TLS 1.3 post-quantum key agreement" inside the chrome flags fixes the problem but only one my client computer.

Is this a problem depending on the relative old OpenSSL version (3.0.15) included in Nginx for Windows (1.27.2)?

**Comment by Maryna-f5** [comment]
This discussion has been inactive for more than 90 days, closing it.
